core/extrinsics_submission_service/extrinsic_submission_proxy.hpp

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_EXTRINSICS_SUBMISSION_SERVICE_EXTRINSIC_SUBMISSION_PROXY_HPP
```

Comment:
```suggestion
#ifndef KAGOME_EXTRINSIC_SUBMISSION_PROXY_HPP
```

Reply:
who looks at it? this is generated automatically following common rules, I don't think we should change it manually.